index.html

```diff
@@ -83,6 +83,7 @@ <h2>Who?</h2>
             <li><a title="Answer Overflow" href="https://github.com/AnswerOverflow/AnswerOverflow" ><img src="./logos/answer-overflow.svg" alt="Answer Overflow logo" /></a></li>
             <li><a title="CodeCrafters" href="https://github.com/codecrafters-io/frontend" ><img src="./logos/codecrafters.svg" alt="CodeCrafters logo"/></a></li>
             <li><a title="Codecov" href="https://github.com/codecov/self-hosted" ><img src="./logos/codecov.svg" alt="Codecov logo" /></a></li>
+            <li><a title="Convex" href="https://convex.dev" ><img src="./logos/convex.svg" alt="Convex logo" /></a></li>
```

Comment:
You up for using https://github.com/get-convex/convex-backend here?

Reply:
Yeah that's also reasonable!

Reply:
```suggestion
            <li><a title="Convex" href="https://github.com/get-convex/convex-backend" ><img src="./logos/convex.svg" alt="Convex logo" /></a></li>
```